src/RenderWebGL.js

```diff
@@ -940,27 +907,47 @@ class RenderWebGL extends EventEmitter {
      * could possibly intersect the given bounds.
      * @param {int} drawableID - ID for drawable of query.
      * @param {Array<int>} candidateIDs - Candidates for touching query.
-     * @param {Rectangle} bounds - Bounds to limit candidates to.
-     * @return {?Array<int>} Filtered candidateIDs, or null if none.
+     * @return {?Array< {id, drawable, intersection} >} Filtered candidates with useful data.
```

Comment:
I wasn't sure how to best annotate this, anyone got a help?

Reply:
I usually end up using `@typedef` in this kind of situation. A good example to check out is `extension-support/define-messages.js` in scratch-vm. That might be overkill for this situation, but it's certainly an option.